coord/src/rmr/conn.c

```diff
@@ -207,7 +225,6 @@ int MRConnManager_ConnectAll(MRConnManager *m) {
   dictEntry *entry;
   while ((entry = dictNext(it))) {
     MRConnPool *pool = dictGetVal(entry);
-    if (!pool) continue;
```

Comment:
why removing?

Reply:
At first I added it to the new functions as well, but after some investigation there is no case of the dictionary iterator returning an entry with no pool. This line here is a leftover from when we used triemaps for this job, and it seems like a common pattern to do with triemaps in other places in the project.